src/main/java/org/broadinstitute/dsm/util/KitUtil.java

```diff
@@ -141,7 +141,9 @@ public static void createLabel(List<KitRequestCreateLabel> kitsLabelTriggered) {
                         KitRequestShipping.updateRequest(kitLabelTriggered, ddpParticipant, kitLabelTriggered.getKitTyp(), kitLabelTriggered.getKitRequestSettings());
                     }
                     else {
-                        KitRequestShipping.deactivateKitRequest(kitLabelTriggered.getDsmKitRequestId(), "Participant not found", null, "System");
+                        KitRequestShipping.deactivateKitRequest(Long.parseLong(kitLabelTriggered.getDsmKitRequestId()), "Participant not found",
+                                null,
+                                "System");
```

Comment:
I think we should change that to all caps "SYSTEM" the created_bu for kit request is that way.